### PR TITLE
Text and typo fixes

### PR DESCRIPTION
**Pull request by mmattel**
As the title says...

### CODE REVIEW

**Comment by mmattel** [comment]
I will double check "shard"...
Just to be sure

**Comment by EParzefall** [comment]
> When I first saw "shard" being used, I thought that you must have been somehow trying to describe a user-backend that has many users, and the users are split across multiple "database places", splitting the "user table" into shards. There were multiple mentions of "sharding" and "shard".
> 
> If you really meant "sharing" and "shares" then fine - this PR is a good correction.

Sharding and shard is correct in this case. Data sets split up and distributed... 
See: https://www.techtarget.com/searchoracle/definition/sharding

File: modules/ROOT/pages/deployment/extensions/ocdav.adoc [hunk 1]
```diff
@@ -27,25 +27,25 @@ include::partial$deployment/extensions/beta-statement.adoc[]
  
 === General View
 
-A PROPFIND finds its way to a storage provider like this. While this is a simplification to get an understanding of what needs to go where, there are several places where sharding can happen.
+A PROPFIND finds its way to a storage provider like in the image shown below. While this is a simplification to get an understanding of what needs to go where, there are several places where sharing can happen.
```
Comment (EParzefall) [inline modules/ROOT/pages/deployment/extensions/ocdav.adoc:30]:
```suggestion
A PROPFIND finds its way to a storage provider like in the image shown below. While this is a simplification to get an understanding of what needs to go where, there are several places where sharding (splitting and distributing) can happen.
```

File: modules/ROOT/pages/deployment/extensions/ocdav.adoc [hunk 2]
```diff
@@ -27,25 +27,25 @@ include::partial$deployment/extensions/beta-statement.adoc[]
...
 
 image::deployment/extensions/ocdav/mermaid-ocdav-1.svg[width=600]
 
 === Proxy-based User Routing
 
-The Infinite Scale proxy authenticates requests an can forward requests to different backends, depending on the logged in user or cookies. For example multiple ocdav services can be configured to shard users based on username or affiliation.
+The Infinite Scale proxy authenticates requests and can forward requests to different backends, depending on the logged in user or cookies. For example, multiple `ocdav` services can be configured to share users, based on username or affiliation.
```
Comment (EParzefall) [inline modules/ROOT/pages/deployment/extensions/ocdav.adoc:36]:
```suggestion
The Infinite Scale proxy authenticates requests and can forward requests to different backends, depending on the logged in user or cookies. For example, multiple `ocdav` services can be configured to shard users, based on username or affiliation.
```

**Review by EParzefall — APPROVED** [review]
2 things to change back.

**Review by phil-davis — APPROVED** [review]

**Pull request merged**